docs/components/dld.md

```diff
@@ -0,0 +1 @@
+::: extra.components.DelayLineDetector
```

Comment:
This file should be added to the nav section in `mkdocs.yml`: https://github.com/European-XFEL/EXtra/blob/master/mkdocs.yml#L19
And the component index: https://github.com/European-XFEL/EXtra/blob/master/docs/components/index.md

Reply:
Thanks, done in a534bb0